config/sync/core.extension.yml

```diff
@@ -23,7 +23,6 @@ module:
   cookie_content_blocker: 0
   cookie_content_blocker_media: 0
   csp: 0
-  ctools: 0
```

Comment:
Ctools finally going!? Hooray :)

Reply:
Yeah, pathauto was the only module with a dependency and that's now been sorted.